packages/moonstone/Scrollable/ScrollButtons.js

```diff
@@ -144,6 +148,16 @@ class ScrollButtons extends Component {
 		this.prevButtonRef = React.createRef();
 	}
 
+	componentDidMount () {
+		this.nextButtonRef.current.addEventListener('keydown', this.onKeyDownNext);
```

Comment:
It looks safe to add event listeners to scroll buttons on mounting only for our components, but I'm not sure it would be okay not to (re-)add listeners on updating in every single case. I expect there will be no DOM tree change on updating, but maybe we're able to add them again for sure since a redundant handler will be ignored. Do we need to do?

Reply:
Is there something that could potentially change in an update - like the ref/dom or something we would need to account for? I may just be unfamiliar with the life-cycle of this component, but I would expect if the dom isn't expected to change, there would be no need to register a dom event after every update.

Reply:
> Is there something that could potentially change in an update - like the ref/dom or something we would need to account for?

No, I don't think either. I'm just worrying _what if later_. Let it go as is.